bigquery/bigquery_storage_quickstart/main.go

```diff
@@ -224,10 +227,31 @@ func processStream(ctx context.Context, client *bqStorage.BigQueryReadClient, st
 			if err == io.EOF {
 				return nil
 			}
-			if err != nil {
-				retries++
-				if retries >= retryLimit {
-					return fmt.Errorf("processStream retries exhausted: %v", err)
+			if err == nil {
```

Comment:
Can you reorder this so the `err==nil` case is last?  As it stands, the logic checks for io.EOF, no error, then generic error (where we inspect for the exhaustion/retryinfo).

Reply:
Done.